src/lib/apputils/net-server.c

```diff
@@ -889,9 +893,15 @@ setup_addresses(struct socksetup *data)
                                  _("Failed setting up a %s socket (for %s)"),
                                  bind_type_names[addr.type],
                                  paddr(ai->ai_addr));
-                goto cleanup;
+                if (ret != EAFNOSUPPORT)
```

Comment:
Maybe?  I'm not sure it's wrong to log that since it points to dubious system behavior.  (I think what's going on is that the system decided not to load the IPv6 kernel module because there are no IPv6 kernel interfaces, but it doesn't have a mechanism to suppress returning IPv6 addresses from getaddrinfo(NULL, ...).)  The code gets more complicated if we try to log an EAFNOSUPPORT error only if we get it for all getaddrinfo results, but not if we get it for some.